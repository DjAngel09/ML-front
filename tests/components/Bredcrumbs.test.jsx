const { render } = require("@testing-library/react")
const { Bredcrumbs } = require("../../src/components")

describe('Testing in Bredcrumbs components', () => {

    const categories = [
        {
            "id": "MLA1144",
            "name": "Consolas y Videojuegos"
        },
        {
            "id": "MLA438578",
            "name": "Accesorios para Consolas"
        },
        {
            "id": "MLA373767",
            "name": "Para Xbox"
        },
        {
            "id": "MLA121980",
            "name": "Xbox One"
        },
        {
            "id": "MLA121989",
            "name": "Controles y Joysticks"
        },
        {
            "id": "MLA121995",
            "name": "Joysticks"
        }
    ]

    test('Must match the snapshot', () => {

       const { container } = render( <Bredcrumbs categories={categories}/> );
       expect( container ).toMatchSnapshot();

    });

})