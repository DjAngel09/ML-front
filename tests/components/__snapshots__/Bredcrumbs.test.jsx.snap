// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing in Bredcrumbs components Must match the snapshot 1`] = `
<div>
  <nav
    aria-label="breadcrumb"
    class="nav pt-16 pb-16 "
  >
    <ol
      class="breadcrumb"
    >
      <li
        class="breadcrumb-item"
      >
        Consolas y Videojuegos
      </li>
      <li
        class="breadcrumb-item"
      >
        Accesorios para Consolas
      </li>
      <li
        class="breadcrumb-item"
      >
        Para Xbox
      </li>
      <li
        class="breadcrumb-item"
      >
        Xbox One
      </li>
      <li
        class="breadcrumb-item"
      >
        Controles y Joysticks
      </li>
      <li
        class="breadcrumb-item"
      >
        Joysticks
      </li>
    </ol>
  </nav>
</div>
`;
